### PR TITLE
Update DPoS Sent Votes endpoint

### LINKED ISSUE #1348: Update DPoS Sent Votes endpoint
### Description

Update the following endpoint.

*Current:*
```
HTTP GET /api/v3/dpos/votes/sent
RPC  get.dpos.votes.sent
```

*Expected:*
```
HTTP GET /api/v3/pos/stakes
RPC  get.pos.stakes
```

### Acceptance Criteria

- The endpoints are updated as expected above
- The response body is up-to-date
- All the methods, variables, files are renamed and refactored appropriately
- Swagger is up-to-date
- Integration tests are up-to-date

### Additional information
*Response:*

200 OK
```jsonc
{
  "data": {
    "account": {
      "address": "lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu",
      "publicKey": "aq02qkbb35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu", // Optional
      "name": "genesis_56", // Optional
    },
    "stakes": [ // prev: votes
      {
        "validatorAddress": "lsk24cd35u4jdq8szo3pnsqe5dsxwrnazyqqqg5eu",
        "amount": "1081560729258"
        "name": "liskhq" // Optional
      }
    ]
  },
  "meta": {
    "count": 8,
    "offset": 0,
    "total": 8
  }
}
```

### PR DESCRIPTION
**Pull request by priojeetpriyom**
### What was the problem?
This PR resolves #1348

### How was it solved?
- [x] The endpoints are updated as expected above
- [x] The response body is up-to-date
- [x] All the methods, variables, files are renamed and refactored appropriately
- [x] Swagger is up-to-date
- [x] Integration tests are up-to-date
- [x] Update posRewardsLocked endpoint tests

### How was it tested?
- [x] Local

### CODE REVIEW

File: tests/integration/api_v3/http/posStakes.test.js [hunk 1]
```diff
@@ -21,59 +21,54 @@ const baseUrlV3 = `${baseUrl}/api/v3`;
...
 		do {
 			// eslint-disable-next-line no-await-in-loop
 			response = await api.get(`${baseUrlV3}/dpos/delegates?limit=1`);
 		} while (!response.data);
-		[refDelegate] = response.data;
+		[refValidator] = response.data;
 	});
 
 	describe(`GET ${endpoint}`, () => {
 		it('Returns list of votes when requested for existing account by address', async () => {
```
Comment (nagdahimanshu) [inline tests/integration/api_v3/http/posStakes.test.js:44]:
```suggestion
		it('Returns list of stakes when requested for existing account by address', async () => {
```

File: tests/integration/api_v3/http/posStakes.test.js [hunk 2]
```diff
@@ -21,59 +21,54 @@ const baseUrlV3 = `${baseUrl}/api/v3`;
...
-			if (refDelegate.name) {
-				const response = await api.get(`${endpoint}?name=${refDelegate.name}`);
-				expect(response.data).toMap(voteSchema);
-				expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
-				expect(response.data.votes.length).toBeLessThanOrEqual(10);
-				expect(response.meta).toMap(metaSchema);
-			}
 		});
 
 		it('Returns list of votes when requested for existing account by address and limit=5', async () => {
```
Comment (nagdahimanshu) [inline tests/integration/api_v3/http/posStakes.test.js:51]:
```suggestion
		it('Returns list of stakes when requested for existing account by address and limit=5', async () => {
```

File: tests/integration/api_v3/http/posStakes.test.js [hunk 3]
```diff
@@ -21,59 +21,54 @@ const baseUrlV3 = `${baseUrl}/api/v3`;
...
-			const response = await api.get(`${endpoint}?address=${refDelegate.address}&limit=5`);
-			expect(response.data).toMap(voteSchema);
+			const response = await api.get(`${endpoint}?address=${refValidator.address}&limit=5`);
+			expect(response).toMap(stakeResponseSchema);
 			expect(response.data.votes.length).toBeGreaterThanOrEqual(1);
 			expect(response.data.votes.length).toBeLessThanOrEqual(5);
-			expect(response.meta).toMap(metaSchema);
 		});
 
 		it('Returns list of votes when requested for existing account by address, limit=5 and offset=1', async () => {
```
Comment (nagdahimanshu) [inline tests/integration/api_v3/http/posStakes.test.js:58]: Please update the test description at all places
```suggestion
		it('Returns list of stakes when requested for existing account by address, limit=5 and offset=1', async () => {
```

File: tests/integration/api_v3/rpc/posStakes.test.js [hunk 4]
```diff
@@ -19,68 +19,63 @@ const { request } = require('../../../helpers/socketIoRpcRequest');
...
 		let response;
 		do {
 			// eslint-disable-next-line no-await-in-loop
 			response = await request(wsRpcUrl, 'get.dpos.delegates', { limit: 1 });
 		} while (!response.result);
-		[refDelegate] = response.result.data;
+		[refValidator] = response.result.data;
 	});
 
 	it('Returns list of votes when requested for existing account by address', async () => {
```
Comment (nagdahimanshu) [inline tests/integration/api_v3/rpc/posStakes.test.js:43]: Same for RPC tests as well
```suggestion
	it('Returns list of stakes when requested for existing account by address', async () => {
```

**Review by nagdahimanshu — CHANGES_REQUESTED** [review]

File: services/blockchain-indexer/methods/dataService/controllers/pos.js [hunk 5]
```diff
@@ -67,37 +67,37 @@ const getPoSUnlocks = async params => {
...
+	if (response.data) stakers.data = response.data;
+	if (response.meta) stakers.meta = response.meta;
 
-	return votesReceived;
+	return stakers;
 };
 
-const getVotesSent = async params => {
-	const votesSent = {
+const getStakes = async params => {
```
[the remark this answers is not in the document]
Comment (sameersubudhi) [reply services/blockchain-indexer/methods/dataService/controllers/pos.js:83]: We can ignore this change. I've updated PR #1371 to be consistent (getPoSStakers -> getStakers).

**Review by sameersubudhi — APPROVED** [review]
Works fine!

The tests still need to be updated. This will be done in the future as part of the epic: https://github.com/LiskHQ/lisk-service/issues/1370.

**Review by nagdahimanshu — APPROVED** [review]
LGTM 👍

**Pull request merged**